config/cesm/machines/config_machines.xml

```diff
@@ -289,7 +289,7 @@ This allows using a different mpirun command to launch unit tests
     <NODENAME_REGEX>.*.cheyenne.ucar.edu</NODENAME_REGEX>
     <!-- MPT sometimes timesout at model start time, the next two lines cause
     case_run.py to detect the timeout and retry FORCE_SPARE_NODES times -->
-    <MPIRUN_RETRY_REGEX>MPT: xmpi_net_accept_timeo/accept() timeout</MPIRUN_RETRY_REGEX>
+    <MPIRUN_RETRY_REGEX>MPT: Launcher network accept (MPI_LAUNCH_TIMEOUT) timed out</MPIRUN_RETRY_REGEX>
```

Comment:
This is an unrelated machine change - with the change of mpt on cheyenne the message text has changed.